src/software/firmware/srcs/blower.cpp

```diff
@@ -50,5 +50,4 @@ int Blower::getSpeed() { return m_speed; }
 
 void Blower::stop() {
     actuator->setCaptureCompare(timerChannel, BlowerSpeed2MicroSeconds(0), MICROSEC_COMPARE_FORMAT);
-    m_speed = 0;
```

Comment:
We don't set m_speed to 0 because we want to keep the latest ajustment of blower speed. 
The real stop is `actuator->setCaptureCompare(timerChannel, BlowerSpeed2MicroSeconds(0), MICROSEC_COMPARE_FORMAT);`

Reply:
👌